lib/options.ts

```diff
@@ -34,6 +34,7 @@ export class Options extends commonOptionsLibPath.OptionsBase {
 			sendPush: { type: OptionType.Boolean },
 			sendEmail: { type: OptionType.Boolean },
 			group: { type: OptionType.Array },
+			isMandatory: { type: OptionType.Boolean},
```

Comment:
Do we need to be prefixed `is` or it can be just `mandatory`? For example `sendPush` is not `shouldSendPush`.